src/UI/EditorGroupView.re

```diff
@@ -210,25 +158,15 @@ let make = (~state: State.t, ~windowId: int, ~editorGroup: EditorGroup.t, ()) =>
             onDimensionsChanged={_ => ()}
             onScroll
             theme
-            rulers
-            showLineNumbers
             editorFont={state.editorFont}
             mode
-            showMinimap
-            maxMinimapCharacters
-            matchingPairsEnabled
             bufferHighlights={state.bufferHighlights}
             bufferSyntaxHighlights={state.syntaxHighlights}
             diagnostics={state.diagnostics}
             completions={state.completions}
             tokenTheme={state.tokenTheme}
             definition={state.definition}
-            shouldRenderWhitespace
-            showMinimapSlider
-            hoverDelay
-            isHoverEnabled
-            shouldRenderIndentGuides
-            shouldHighlightActiveIndentGuides
+            config={state.config.settings}
```

Comment:
The configuration collection has no application-specific type information, and so could be even be used for generic components.

Reply:
Excellent 👌